src/Emitter.ts

```diff
@@ -100,22 +113,32 @@ export class Emitter
 
     private _createTreeNodes(docs: TAnyDoclet[])
     {
+        debug(`Emitter._createTreeNodes()`);
+
         for (let i = 0; i < docs.length; ++i)
         {
             const doclet = docs[i];
 
             if (doclet.kind === 'package' || this._ignoreDoclet(doclet))
+            {
+                debug(`Emitter._createTreeNodes(): skipping ${docletDebugInfo(doclet)} (package or ignored)`, doclet);
                 continue;
+            }
 
             if (!this._treeNodes[doclet.longname])
             {
+                debug(`Emitter._createTreeNodes(): adding ${docletDebugInfo(doclet)} to this._treeNodes`);
                 this._treeNodes[doclet.longname] = { doclet, children: [] };
+            } else {
```

Comment:
nit: Don't use K&R style braces.

Reply:
Got it. I've checked in the rest of the code.